panther_manager/README.md

```diff
@@ -44,129 +44,146 @@ echo $USERNAME 'ALL=(ALL) NOPASSWD: /sbin/poweroff, /sbin/reboot, /sbin/shutdown
+- `LOW_BATTERY_ANIM_ID` [*unsigned*, value: **6**]: animation ID constant obtained from `panther_msgs::LEDAnimation::LOW_BATTERY`.
+- `CRITICAL_BATTERY_ANIM_ID` [*unsigned*, value: **7**]: animation ID constant obtained from `panther_msgs::LEDAnimation::CRITICAL_BATTERY`.
+- `BATTERY_STATE_ANIM_ID` [*unsigned*, value: **8**]: animation ID constant obtained from `panther_msgs::LEDAnimation::BATTERY_STATE`.
+- `CHARGING_BATTERY_ANIM_ID` [*unsigned*, value: **9**]: animation ID constant obtained from `panther_msgs::LEDAnimation::CHARGING_BATTERY`.
+- `POWER_SUPPLY_STATUS_UNKNOWN` [*unsigned*, value: **0**]: power supply status constant obtained from `sensor_msgs::BatteryState::POWER_SUPPLY_STATUS_UNKNOWN`.
+- `POWER_SUPPLY_STATUS_CHARGING` [*unsigned*, value: **1**]: power supply status constant obtained from `sensor_msgs::BatteryState::POWER_SUPPLY_STATUS_CHARGING`.
+- `POWER_SUPPLY_STATUS_DISCHARGING` [*unsigned*, value: **2**]: power supply status constant obtained from `sensor_msgs::BatteryState::POWER_SUPPLY_STATUS_DISCHARGING`.
+- `POWER_SUPPLY_STATUS_NOT_CHARGING` [*unsigned*, value: **3**]: power supply status constant obtained from `sensor_msgs::BatteryState::POWER_SUPPLY_STATUS_NOT_CHARGING`.
+- `POWER_SUPPLY_STATUS_FULL` [*unsigned*, value: **4**]: power supply status constant obtained from `sensor_msgs::BatteryState::POWER_SUPPLY_STATUS_FULL`.
 
 ### Safety 
  
-Tree responsible for monitoring the Panther robot's state and handling safety measures, such as cooling the robot in case of high CPU or battery temperature.
+A tree responsible for monitoring the Panther robot's state and handling safety measures, such as cooling the robot in case of high CPU or battery temperatures.
 
 <p align="center">
   <img align="center" src="https://github-readme-figures.s3.eu-central-1.amazonaws.com/panther/panther_ros/safety_tree.png">
 </p>
 
 Default blackboard entries:
-- `aux_state` [*bool*, default: **None**]: state of AUX power.
-- `bat_temp` [*double*, default: **None**]: moving average of battery temperature.
-- `cpu_temp` [*double*, default: **None**]: moving average of cpu temperature
-- `driver_temp` [*double*, default: **None**]: moving average of driver temperature, for condition simplification only motor driver with higher temperature is considered.
-- `e_stop_state` [*bool*, default: **None**]: state of emergency stop.
-- `fan_state` [*bool*, default: **None**]: state of fan.
+- `aux_state` [*bool*, default: **false**]: state of AUX power.
+- `bat_temp` [*double*, default: **NaN**]: moving average of battery temperature.
+- `cpu_temp` [*double*, default: **NaN**]: moving average of CPU temperature
+- `driver_temp` [*double*, default: **NaN**]: moving average of driver temperature. Out of the two drivers, the one with the higher temperature is taken into account.
+- `e_stop_state` [*bool*, default: **true**]: state of emergency stop.
+- `fan_state` [*bool*, default: **true**]: state of fan.
```

Comment:
I mentioned this issue in a comment on the bt node.

Reply:
I reverted those changes